packages/core/src/types.ts

```diff
@@ -1952,6 +1952,10 @@ export interface ActorRef<
   /** @internal */
   _processingStatus: ProcessingStatus;
   src: string | AnyActorLogic;
+  on: (
+    emittedEventType: string,
+    handler: (emittedEvent: AnyEventObject) => void
```

Comment:
Nice try with that `AnyEventObject` 🤣 We'll have to introduce `TNotification` in this PR an pass it around and around 😢